### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by flauntingspade4**
Adds documentation, but none for `calculator::boil_off`, due to the possible issue in #48

### CODE REVIEW

**Comment by flauntingspade4** [comment]
Someone may want to check my spelling

**Review by drodil — APPROVED** [review]
Good start!

File: src/calculators/sg_correction.rs [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+//! and the calibration temperature
+
 pub struct SgCorrection;
 
 impl SgCorrection {
+    /// Corrects a specific gravity value with respect to
+    /// a given calibration temperature, measurement
+    /// temperature
+    /// ```
+    /// // rustybeer sg_correction --sg 5 --ct 23 --mt 22
```
Comment (drodil) [inline src/calculators/sg_correction.rs:13]: Here maybe use the function call as an example instead the CLI commans

Comment (flauntingspade4) [reply src/calculators/sg_correction.rs:13]: Good point, I'll go ahead and get rid of it

**Review by drodil — APPROVED** [review]

**Pull request merged**